index.html

```diff
@@ -2370,13 +2370,41 @@ <h3>Unlinkability</h3>
       <section>
         <h3>Selective Disclosure</h3>
 
-        <p class="issue">
-When the contents of a digitally signed payload contains correlatable
-identifiers, those identifiers can be used to track individuals. A static
-digital signature is a correlatable identifier. There are selective
-disclosure digital signature schemes, such as BBS+, that are capable of
-not disclosing correlatable identifiers and ensuring that a different
-but valid digital signature is re-created upon every presentation.
+        <p>
+Selective disclosure is a technique that enables the recipient of a
+previously-signed message (that is, a message signed by its creator) to reveal
+parts of the message without revealing the entire message and without disturbing
+the verifiability of the origin of the parts being revealed. For example,
+selectively disclosing a digital driver's license for the purpose of renting a
+car might involve the entity presenting the license to only reveal the issuing
+authority, license number, birthday, and authorized motor vehicle class. Note
+that in this case, the license number is correlatable information that is being
+shared but some amount of privacy is still being preserved because the driver's
+full name and address are not being shared.
+        </p>
+
+        <p>
+The creator of an initial message that they expect to be selectively disclosed
+needs to use a capable cryptosuite and enable selective disclosure on the
+message in order for a recipient of that message to use the technique on future
+disclosures. The creator might also need to make it mandatory to disclose
+certain parts of a message. The recipient of the message that desires to
+selectively disclose the original message needs to then utilize software that is
+capable of performing selective disclosure. Not all software or cryptosuites are
+capable of providing selective disclosure. An example of a cryptosuite that does
+support selective disclosure is `bbs-2022`.
+        </p>
+
+        <p>
+It is important to realize that selective disclosure does not always mean that
+all aspects of privacy are preserved. It is possible to selectively disclose
+information in a way that does not preserve privacy. For example, one might want
+to disclose the inspection results related to a shipment, which include the
+shipment identifier or lot number, which might have to be correlatable due to
+regulatory requirements. However, disclosure of the entire inspection result
+might not be required as selectively disclosing just the pass/fail status could
+be deemed as adequate. For more information on disclosing information while
+preserving privacy, see Section <a href="#unlinkability"></a>.
```

Comment:
I suggest the removal of the first sentence because it's redundant with the second except for the "it's important to realize" part. I think that part can also be omitted by just stating the fact as the second sentence does.

I also generally think that if we're going to talk about "preserving privacy" it would be better to indicate, specifically, what is being preserved (or not preserved) rather than leaving it up to the reader to figure that out. Even in the example above, some privacy is preserved (the entire inspection result), right? Therefore, I replaced `privacy` with `unlinkability` -- which was introduced in the previous section. It seems that that is what we're talking about here anyway in the example.

```suggestion
It is possible to selectively disclose information in a way that does not preserve
unlinkability. For example, one might want to disclose the inspection results related
to a shipment, which include the shipment identifier or lot number, which might
have to be correlatable due to regulatory requirements. However, disclosure of
the entire inspection result might not be required as selectively disclosing just
the pass/fail status could be deemed as adequate. For more information on
disclosing information while preserving privacy, see
Section <a href="#unlinkability"></a>.
```